lib/rabbitmq/cli/core/helpers.ex

```diff
@@ -187,6 +187,14 @@ defmodule RabbitMQ.CLI.Core.Helpers do
     :net_adm.ping(node) == :pong
   end
 
+  def rabbit_app_running?(%{node: node, timeout: timeout}) do
```

Comment:
Should we make it clear that this function has 3 return types? For example `running | stopped | {error, _}`?